src/components/filters/AttributeFilter/AttributeDropdown.tsx

```diff
@@ -185,16 +168,19 @@ export class AttributeDropdownWrapped extends React.PureComponent<
             isListReady: false,
             listError: null,
             items: [],
-            selection: [],
-            isInverted: true,
+            selection: props.selection || [],
+            prevSelection: null,
+            isInverted: props.isInverted !== undefined ? props.isInverted : !props.selection.length,
```

Comment:
This looks like too much magic. I would expect a default true or false ignoring selection.length. Changing selection then doesn't change state.isInverted.

Reply:
Well, here I would disagree. It is a bit of magic, but well justified. When there is no selection, you naturally expect all the elements to be checked, and thus you need negative/inverted filter with no selection. But when you supply some selection, you naturally expect the filter not to be negative, thus positive/not-inverted with some attribute elements checked.